cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -1025,6 +1028,43 @@ To generate a QR code for multiple end devices:
 			return nil
 		}),
 	}
+	endDevicesExternalJSCommand = &cobra.Command{
+		Use:     "external-join-server [application-id] [device-id]",
+		Aliases: []string{"ext-js"},
+		Short:   "Disassociate and delete the device from Join Server",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			devID, err := getEndDeviceID(cmd.Flags(), args, true)
+			if err != nil {
+				return err
+			}
+			if !config.JoinServerEnabled {
+				return errJoinServerDisabled
+			}
+			js, err := api.Dial(ctx, config.JoinServerGRPCAddress)
+			if err != nil {
+				return err
+			}
+			_, err = ttnpb.NewJsEndDeviceRegistryClient(js).Delete(ctx, devID)
```

Comment:
@htdvisser do we want to do this when the JS address matches what is currently in IS, or do we not care about this?

Reply:
We also check the registered addresses against the configured addresses in other commands, so I think we should also do that here.